src/plugins/expressions/public/plugin.ts

```diff
@@ -51,7 +56,7 @@ export class ExpressionsPublicPlugin implements Plugin<ExpressionsSetup, Express
     setRenderersRegistry(renderers);
     setExpressionsService(expressions);
 
-    const setup = expressions.setup();
+    const setup = expressions.setup(core);
```

Comment:
why do we pass whole `core` ? don't we only need the `getStartServices` fn ?

Reply:
I've corrected the argument type in the `ExpressionsService` not to require the entire core. At this place, there is no need to pass only the `getStartServices` function, but I changed that anyway to avoid any concerns.